card/spec/lib/card/cache_spec.rb

```diff
@@ -76,7 +76,7 @@
 
   describe "with file store" do
     before do
-      cache_path = "#{Decko::Rails.root}/tmp/cache"
+      cache_path = "#{Wagn.root}/tmp/cache"
```

Comment:
This is a spec, but the deeper problem is that neither of those namespaces should be referenced in Card.  Truth is none of them should be referenced in the test and this should come from the class under test directly.  That may just move the problem there, but at least it will be in the right place.